src/Xamarin.Android.Build.Tasks/Linker/MonoDroid.Tuner/MonoDroidProfile.cs

```diff
@@ -19,6 +19,7 @@ protected override bool IsSdk (string assemblyName)
 		{
 			return assemblyName.Equals ("Java.Interop", StringComparison.Ordinal)
 			       || assemblyName.Equals ("Java.Interop.GenericMarshaler", StringComparison.Ordinal)
+			       || assemblyName.Equals ("_Microsoft.Android.Resource.Designer", StringComparison.Ordinal)
```

Comment:
It seems odd that this change is required; shouldn't it be possible to mark `_Microsoft.Android.Resource.Designer.dll` as "trimmable" (I think there's a custom attribute for that?), which would remove the need for this change?

Reply:
There is a spot where `%(ResolvedFileToPublish.IsTrimmable)` is set to `false`, maybe you don't need this line.